spec/schemas/ciag-induction/ciag-induction-created.yml

```diff
@@ -0,0 +1,55 @@
+name: ciag-induction.created
+title: CIAG Induction Created
+summary: A CIAG induction has been created for a Prisoner.
+contentType: application/json
+payload:
+  type: object
+  properties:
+    eventType:
+      type: string
+      example: "ciag-induction.created"
+    version:
+      type: integer
+      example: 1
+    detailUrl:
+      type: string
+      example: "https://learning-and-work-progress.hmpps.service.justice.gov.uk/plan/A0670DZ/view/overview"
+    occurredAt:
+      type: string
+      example: "2023-10-05T12:00:00.000Z"
+    description:
+      type: string
+      example: "CIAG Induction Completed"
```

Comment:
Similar to the comment about `eventType` ... is this an example, or is it the 1 and only 1 description value that we would expect producers of these messages to provide?

I guess the question is what is the purpose of the `description` field? What are we as consumers of the message going to do with it?

Does it matter if the CIAG API produces a message with a description field of `CIAG Induction Completed` or `A CIAG Induction was completed for prisoner A1234BC` or `wibble-wobble` ?

Reply:
I'm not sure what it's used for exactly, but it's in every other message spec in this repo! I'd rather not enforce anything on it.